gems/aws-sdk-core/sig/aws-sdk-core.rbs

```diff
@@ -0,0 +1,57 @@
+module Aws
```

Comment:
Is it best practice to stuff rbs into one file or to have a directory/file structure similar to lib files?

Reply:
There are no specific best practices, and the granularity of files is chosen based on preference.
However, it has been reported that performance issues with Steep may arise if the number of RBS files exceeds 10,000.

https://github.com/soutaro/steep/issues/661

Reply:
Makes sense. I think we should break these out into a similar structure like lib/ then adding more becomes easier to manage and organized.